tests/Unit/ParallelAlgorithms/EventsAndTriggers/CMakeLists.txt

```diff
@@ -19,3 +19,9 @@ add_dependencies(
   ${LIBRARY}
   module_ConstGlobalCache
   )
+
```

Comment:
so let's get this merged to fix the problem, but going forward is the plan to eliminate `add_test_library` as I don't know if I am supposed to add libs to the final argument of that or to `target_link_libraries`...

Reply:
The plan at least is to no longer specify the  `target_link_libraries` as part of the call, they should be added after :)